internal/checker/cloudformation/rules/logging/verify_redshift_cluster_logging_enabled.rego

```diff
@@ -0,0 +1,50 @@
+# METADATA
+# title: "Verify Redshift Cluster Logging is Enabled"
+# description: "Checks if logging is enabled for an AWS Redshift Cluster."
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-redshift-cluster.html
+# custom:
+#   id: CB_CFT_057
+#   severity: MEDIUM
+package lib.cloudformation.CB_CFT_057
+
+import future.keywords.in
+
+resource := "AWS::Redshift::Cluster"
+
+is_valid {
+    some resources in input.Resources
+    resources.Type == resource
+}
+
```

Comment:
update logic for pass
pass[properties] {
    is_valid
    some resources in input.Resources
    resources.Type == resource
    properties := resources.Properties
    "BucketName" in object.keys(properties.LoggingProperties)
}